tuple/sql/postgres/postgres.go

```diff
@@ -3,11 +3,10 @@ package postgres
 import (
 	"strconv"
 
-	_ "github.com/lib/pq"
```

Comment:
Same as for `mysql`: do not remove side-effect imports. They are here for a reason.